pkgs/applications/office/todoman/default.nix

```diff
@@ -67,6 +67,7 @@ buildPythonApplication rec {
     "test_default_command"
     "test_main"
     "test_missing_cache_dir"
+    "test_sorting_fields"
```

Comment:
Should we only do this if the tests are running on darwin?

Reply:
Yes please, it never fails on z86_64-linux AFAICT

Reply:
SG -- updated!